pkg/scheduler/plugins/usage/usage.go

```diff
@@ -43,35 +41,63 @@ const (
    - plugins:
      - name: usage
        arguments:
-          thresholds:
-            CPUUsageAvg.5m: 80
-            MEMUsageAvg.5m: 90
+         usage.weight: 10
+         type: average
+         thresholds:
+           cpu: 70
+           mem: 70
+         period: 10m
 */
```

Comment:
if I want to take both Avg.10m and Avg.30m into account, how to config it?

Reply:
1 firstly
I  think both use AVG and COMMON is not good,so I do not support it
2 secondly 
I  think both use 5m and 10m  is not good,so I do not support it either

by the way
I make a new PR
https://github.com/volcano-sh/volcano/pull/3056

and this PR is closed

Reply:
> I make a new PR

please update commit on the same PR. thanks.